### ISSUE TITLE
SSL for filerun

### ISSUE THREAD

**Issue by manish-fn** [filerun/docker#29]
I have installed filerun through Docker. However since filerun uses port 80, i am unable to install ssl certficates on it.

**Comment by KR0SIV** [comment]
What I did was shut down all services using that port.
Then I started my container mapped to 443 and 80.

From there I exec -it into the container and download/run certbot.
Install is successful.

I then followed this tutorial, basically shut down the container and docker service, manually edit the port mapping, start the service back up and restart the container.

Then just bring your services back up on their respective ports.

https://stackoverflow.com/questions/19335444/how-do-i-assign-a-port-mapping-to-an-existing-docker-container
